CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.4.1)
+project(openGA CXX C)
+set(CMAKE_CXX_STANDARD 11)
+set(THREADS_PREFER_PTHREAD_FLAG ON)
+find_package(Threads REQUIRED)
+find_package(Boost COMPONENTS system filesystem thread chrono REQUIRED)
```

Comment:
Adding any dependency to the project rather than the standard library is what I have tried to avoid. Previously, this project depended on boost. But, I removed this dependency by extra effort. It is hard to convince the users to install dependencies to use this library.

Reply:
This CMakeLists is outdated. I delete the Boost also.